README.md

```diff
@@ -1,16 +1,174 @@
-# Title
+# Programming as Conversation 2: Selection with if...end and Friends
 
 ## Learning Goals
 
--SWBAT 1
+* Write an `if...end` statement
 -SWBAT 2
 
 ## Introduction
 
-## SWBAT 1
+We've covered the **default sequence** Ruby follows when reading and executing
+a Ruby file. Let's teach the sequence to change by using the **selection**
+statement: `if...end`.
 
-## SWBAT 2
+## Write an `if...end` Statement
+
+```ruby
+run_code_inside = true
+puts "Code before if...end"
+if run_code_inside
+  puts "code inside"
+end
+puts "Code after if...end"
+```
+
+Try entering this code in your file and try running it. The output should be
```

Comment:
I think first the student should be instructed to create a file before referring to it here (could continue thread of 'create a file, demo.rb' from previous lesson)